### DISCUSSION TITLE
Accessibility of the generated PDFs

### DISCUSSION THREAD

**Discussion by denonbw** [pdfme/pdfme#260] [General]
Hello
I am currently comparing disappearing libraries for PDF creation.
PDFME looks great, but unfortunately, I cannot find any information regarding the accessibility of the generated PDFs. 
Is it possible to work with tags and structure in PDFME in terms of accessibility?

**Comment by hand-dot** [comment]
at this moment, we can't

**Comment by aidanwyber** [comment]
Is it possible to add any kind of metadata to a pdf at all? Like a language 'EN-US' for example?
